main.tf

```diff
@@ -2,20 +2,20 @@ variable "region" {}
 variable "account" {}
 
 output "primary" {
-    value = "${lookup(var.primary_azs, format(\"%s-%s\", var.account, var.region))}"
+    value = "${lookup(var.primary_azs, format(""\"%s-%s\""", var.account, var.region))}"
```

Comment:
I think it should be changed from `\"` to `"`, because this is what Terraform got changed some versions back and this is exactly what is wrong with this module.